sessionctx/variable/sysvar.go

```diff
@@ -983,7 +983,7 @@ var defaultSysVars = []*SysVar{
 		s.BatchCommit = TiDBOptOn(val)
 		return nil
 	}},
-	{Scope: ScopeGlobal | ScopeSession, Name: TiDBDMLBatchSize, Value: strconv.Itoa(DefDMLBatchSize), Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxUint64, SetSession: func(s *SessionVars, val string) error {
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBDMLBatchSize, Value: strconv.Itoa(DefDMLBatchSize), Type: TypeUnsigned, MinValue: 0, MaxValue: math.MaxInt64, SetSession: func(s *SessionVars, val string) error {
```

Comment:
Then why do you use MaxInt64 here?

Reply:
It looks like an incorrect fix. I'll change it to be `math.MaxInt32` as well.

Reply:
https://github.com/pingcap/tidb/pull/24964/commits/b86105a5cb1d5ec82d9e1ff0ed2a4530e4874f15